### DISCUSSION TITLE
MPT 30B inference on Mac M1

### DISCUSSION THREAD

**Discussion by RonanKMcGovern** [ggml-org/ggml#353] [Ideas]
Is it realistic to try and get inference running on Mac M1 with similar results quality as on a GPU?

I find 7B and 13B models are not good enough to get working well with functions. Also, I like that MPT has extendable context compared to Falcon and llama.

If I were to try and get MPT 30B running, can I bootstrap using work from llama? Thanks

**Comment by ggerganov** [comment]
This is now supported in [llama.cpp](https://github.com/ggerganov/llama.cpp) : https://github.com/ggerganov/llama.cpp/pull/3417
